text/0000-target-tier-policy.md

```diff
@@ -461,6 +461,9 @@ including the infrastructure team in the RFC proposing the target.
 - The target must build and pass tests reliably in CI.
   - The target must not disable an excessive number of tests or pieces of tests
     in the testsuite in order to do so. This is a subjective requirement.
+  - If the target does not have host tools support, or if the target has low
+    performance, CI may cross-compile the testsuite from another platform, and
+    then run the compiled tests natively.
```

Comment:
Hmm, I'm not sure whether we should do the test cross-compilation for host targets, regardless of the performance of doing so. If the compiler is so slow that CI is not practical that's a problem.

Reply:
@pietroalbini I added some further discussion on this point, and also expanded the requirements for both tier 1 and tier 1 with host tools to better address such performance considerations.